src/firefly/js/visualize/ui/ColorPanelReducer.js

```diff
@@ -252,18 +282,17 @@ var getFieldInit= function() {
             label: 'Samples per line:'
         },
 
-        beta: {
-            fieldKey: 'beta',
-            value: NaN,
-            validator: Validate.isPositiveFiniteNumber.bind(null, 'Beta'),
-            tooltip: 'an arbitrary "softness"',
-            label: 'Beta:'
+        asinhQ: {
+            fieldKey: 'asinhQ',
+            validator: Validate.floatRange.bind(null, 0, 30, 4, 'Q'),
```

Comment:
I suggest to change the 30 to 100 or even 1000. The slider prevents a user from going over 25 right now. 

This limit only applies to the UI and not to values passed through the JS API, right?